util/dvsim/dvsim.py

```diff
@@ -588,10 +576,10 @@ def main():
     if args.items != []:
         # Create deploy objects.
         cfg.create_deploy_objects()
-        cfg.deploy_objects()
+        results = cfg.deploy_objects()
```

Comment:
I imagined that a `FlowCfg` was the "configuration for a flow" (which doesn't include the results of running it). If you believe it would be cleaner to store those results on the object, I'm happy to do that instead. What do you think?